website/content/v1.10/introduction/getting-started.md

```diff
@@ -1,323 +1,229 @@
 ---
-title: Getting Started
+title: Getting Started with a Local Talos Cluster 1.10
```

Comment:
let's please not put the version anywhere, as it makes duplicating docs for the next release too hard

We have `{{< version >}}` or similar template variable if you really need a version